src/OpenTelemetry/Trace/TracerProviderSdk.cs

```diff
@@ -97,6 +97,10 @@ internal class TracerProviderSdk : TracerProvider
                         return;
                     }
 
+                    // Spec says IsRecording must be true once span ends.
+                    // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#isrecording
```

Comment:
I guess the spec says it should be false?

Reply:
Oh my bad! (time to stop working on Friday :D )

Reply:
I would vote for leaving this block as commented out:
1. `IsAllDataRequested` has a slightly different semantic than `IsRecording`.
2. We don't see a use case, except for burning extra CPU cycle.

Reply:
Agree with @reyang, seems a bit silly to do this 😜